lib/handler/mruby.c

```diff
@@ -47,8 +47,7 @@
 
 void h2o_mruby__abort_exc(mrb_state *mrb, const char *mess, const char *file, int line)
 {
-    h2o_error_printf("%s at file: \"%s\", line %d: %s\n", mess, file, line, RSTRING_PTR(mrb_inspect(mrb, mrb_obj_value(mrb->exc))));
-    abort();
+    h2o_fatal("%s at file: \"%s\", line %d: %s\n", mess, file, line, RSTRING_PTR(mrb_inspect(mrb, mrb_obj_value(mrb->exc))));
```

Comment:
Should we do something like `h2o__fatal(file, line, "fatal error: %s, %s\n", mess, RSTRING_PTR(...))` here?

Reply:
Agree.